cmd/ddev/cmd/snapshot_restore_test.go

```diff
@@ -44,7 +45,7 @@ func TestCmdSnapshotRestore(t *testing.T) {
 
 	// Try interactive command
 	// Doesn't seem to work without pty, 2021-12-14
-	//if runtime.GOOS != "windows" {
+	//if !nodeps.IsWindows() {
 	//	out, err = exec.RunCommand("bash", []string{"-c", "echo -nq '\n' | " + DdevBin + " snapshot restore"})
 	//	assert.NoError(err)
 	//	assert.Contains(out, "Restored database snapshot")
```

Comment:
I can't imagine that these lines are useful. If you push some more code, remove.